src/wrappers/contextMenus.css

```diff
@@ -7,9 +7,9 @@ body {
 
 /* This is the class name defined by react-contexify */
 .contexify {
-  @apply z-modals min-w-min p-1;
+  @apply z-modals min-w-min p-0;
 }
 
-.contexify .contexify_itemContent {
-  @apply px-2 py-0.5;
+.contexify_itemContent {
+  @apply p-3;
```

Comment:
This was made to prevent the "+ Add" popup to look like 

![image](https://github.com/refstudio/refstudio/assets/174127/f13f2c45-78f0-4f29-99d3-9599c8b72339)

And instead like this

![image](https://github.com/refstudio/refstudio/assets/174127/521769b1-9fe4-4335-90a4-74e0cd23490b)

The side effect of this change is that now all the other context menu items also look the same:

![image](https://github.com/refstudio/refstudio/assets/174127/2cb15385-af2c-4f1b-a201-58ac0f1cda02)

Instead of the original

![image](https://github.com/refstudio/refstudio/assets/174127/2e4184e5-e096-4f71-900d-eb458d319193)


This is another reason to reimplement the popup.

Reply:
This was intentional. There doesn't seem to be official "design" for the context menus and with the redesign more white space seemed desirable to me. @hammer what do you think?

Reply:
The difference is marginal to my eye from these images, maybe it would be easier to distinguish on a live app. I’m traveling today so don’t block on me, just pick the easiest path and file an issue to address later.

Reply:
@jvoytek, we can go with your change as we don't have a design for the context menu and we also don't have a button dropdown ATM. But we need to revisit this with the design team in the future.